### PR TITLE
Added utils from Build to export.

### PR DESCRIPTION
**Pull request by pomek**
Fixes: #10 

This module is needed for collect paths to CKEditor's dependencies.

### CODE REVIEW

**Comment by szymonkups** [comment]
If some method is needed not only in builder I think they should be in `ckeditor5-dev-utils` rather than be exported by builder.

From F2F discussion with @pomek, the needed method is `getPackages( rootDir )` which gets paths to `ckeditor5-` packages (and resolves symlinks too). I think it is fine to move it to dev utils.

**Comment by pomek** [comment]
A task about `getPackages` method was moved to #3.

**Comment by Reinmar** [comment]
> If some methods are needed not only by builder I think they should be in ckeditor5-dev-utils rather than be exported by builder.

This means that utils will get a lot of specific functions, which makes sense only for all form of compiling. @pomek has just made https://github.com/ckeditor/ckeditor5-dev-utils/pull/12 and this looks bad. He had to move there a function which was required by another function which makes sense only in context of compilation. Both of them should be here, in this package and all of them can be exposed by this package.

File: lib/index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@
 module.exports = ( config ) => {
 	return {
 		builder: require( './builder' )( config ),
+		builderUtils: require( './builder/utils' ),
```
Comment (Reinmar) [inline lib/index.js]: I'd make it a simple `utils`.

**Pull request closed without merging**